src/Pages/Home/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react'
+import { SongCard } from '../../Components/SongCard'
+import { ArtistCard } from '../../Components/ArtistCard'
+import { GigCard } from '../../Components/GigCard'
+import { APIErrorCard } from '../../Components/APIError'
+import { RecordingButton } from '../../Components/RecordingButton'
+import { SongNotFound } from '../../Components/SongNotFound'
+import { Menu } from '../../Components/Menu'
+import { useCookies } from "react-cookie";
+import { useNavigate } from 'react-router-dom';
+
+// import AuthCallback  from '../AuthCallback'
+
+export default function Home() {
+  const [title, setTitle] = useState('')
+  const [album, setAlbum] = useState('')
+  const [artist, setArtist] = useState('')
+  const [artwork, setArtwork] = useState()
+  const [songNotFound, setSongNotFound] = useState(false)
+  const [cardState, setCardState] = useState('songCard')
+  const [APIError, setAPIError] = useState(false)
+  const [loading, setLoading] = useState(false)
+  const [cookies, setCookie, removeCookie] = useCookies(["accessToken"])
+  let navigate = useNavigate();
+
+  // const [spotifyArtistID, setSpotifyArtistID] = useState("")
```

Comment:
remove

Reply:
Comments are prep for the spotify stuff we'll be working on this week